### DISCUSSION TITLE
Detector: Object

### DISCUSSION THREAD

**Discussion by Dvalin21** [doitandbedone/ispyagentdvr-docker#619] [Q&A] [answered]
Is it possible to set Object Detection under "Detector" to stop detecting stationary items?  I know there is a setting under the AI Object Detection, but I don't want it to be triggered because of an alert or motion. I would like for it to be triggered as the AI Object Detection is allow to do a constant scan, but there isn't a selection for this.

**Comment by ispysoftware** [comment, marked as answer]
you can set AI object recognition to run on an interval instead Then under objects to look for check "Ignore Static Objects"
That way you don't need a detector at all.

**Reply by Dvalin21** [reply]
Thank you!
